ComplexParser.py

```
@@ -1,89 +1,479 @@
-import os
+import os, re
 from jinja2 import Environment, FileSystemLoader
 from util import paths
+from STParser import *
```

Ruff (0.12.2)

`from STParser import *` used; unable to detect undefined names

(F403)